docs/dev/jobs.md

```diff
@@ -121,7 +121,6 @@ from nautobot_ssot.contrib import NautobotAdapter
 
 from your_ssot_app.models import DiffSyncDevice, DiffSyncPrefix, DiffSyncIPAddress
 
```

Comment:
Why was the extra line removed here? There are supposed to be two lines between imports and the top most function/class.